### PR TITLE
fix: npm run docs on Windows (#3973)

### LINKED ISSUE #3973: `npm run docs` doesn't work on Windows
```
C:\Users\xmr\Desktop\cli>npm run docs

> netlify-cli@8.6.12 docs
> node ./site/scripts/docs.js

Docs updated!
No files matched pattern [
  'C:\\Users\\xmr\\Desktop\\cli\\README.md',
  'C:\\Users\\xmr\\Desktop\\cli\\docs\\**\\**.md'
]
```

I suspect it's due to the backslashes.

### PR DESCRIPTION
**Pull request by iib0011**
#### Summary

Fixes #3973

### CODE REVIEW

**Review by erezrokah — CHANGES_REQUESTED** [review]
Thanks @iib0011, I left a comment

File: site/scripts/docs.mjs [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
 import markdownMagic from 'markdown-magic'
 import stripAnsi from 'strip-ansi'
 
 import { generateCommandData } from './generate-command-data.mjs'
+import {normalizeBackslash} from "../../src/lib/path.js";
 
 const rootDir = fileURLToPath(new URL('../..', import.meta.url))
-const markdownFiles = [join(rootDir, 'README.md'), join(rootDir, 'docs/**/**.md')]
+const isWindows = platform === 'win32'
+const markdownFiles = [
```
[the remark this answers is not in the document]
Comment (iib0011) [reply site/scripts/docs.mjs:14]: No, it wouldn't since there is a condition on the platform

[the remark this answers is not in the document]
Comment (erezrokah) [reply site/scripts/docs.mjs:14]: It the condition required? Will it work for all platforms without the condition?

[the remark this answers is not in the document]
Comment (iib0011) [reply site/scripts/docs.mjs:14]: the condition is required for windows users

[the remark this answers is not in the document]
Comment (erezrokah) [reply site/scripts/docs.mjs:14]: Thanks for the additional information, let me clarify as I had a typo:
1. For windows, running `normalizeBackslash` will fix the issue
2. For non-windows, running `normalizeBackslash` will not have an impact, so we can keep it

Is that correct?

[the remark this answers is not in the document]
Comment (iib0011) [reply site/scripts/docs.mjs:14]: I see now. Let me correct it

[the remark this answers is not in the document]
Comment (iib0011) [reply site/scripts/docs.mjs:14]: fixed

[the remark this answers is not in the document]
Comment (erezrokah) [reply site/scripts/docs.mjs]: Added another commit in https://github.com/netlify/cli/pull/4319/commits/2011c3101ff3bbe88e74b1ccbdb212c8fdaddadb.

Can you let me know if that works for you on Windows?

[the remark this answers is not in the document]
Comment (iib0011) [reply site/scripts/docs.mjs]: Yes it works

**Pull request merged**